blog/2024-01-09-introducing-bidirectional-github-sync/index.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: Bidirectional GitHub Sync is now here!
+description: Watch our latest office hours live stream where we cover new features including Stately Inspect, GitHub Sync, Sources, and our roadmap for 2024.
+tags: [stately, studio, GitHub, sync, git]
+authors: [kevin]
+date: 2024-01-09
+slug: 2024-01-09-introducing-bidirectional-github-sync
+# image: ''
+---
+
+The team at Stately is proud to release an exciting new feature to kick off 2024, [GitHub Sync with Stately Studio](https://stately.ai/docs/import-from-github)! This feature has been highly requested by users and it prompted us to initially launch GitHub Import as a Pro feature over a year ago. But GitHub Sync goes beyond just importing as it enables a completely bidirectional workflow between GitHub repositories and Stately Studio. View the [short demo of GitHub Sync](https://www.youtube.com/watch?v=x-F1TPjz_lI&t=318s) from our Office Hours last month.
+
+{/* truncate */}
+
+![Connect a project to a GitHub repository by clicking the Connect GitHub button.](connect-github-button.png)
+
+### Manual syncing is so 2023
```

Comment:
@laurakalbag This and the next 2 small sections were originally under a heading related to the "why", whereas the following group of sections is under the "What...". Do we need an actual heading for the first group of "why"?

Also, for all of the "how" I figured we could link to the docs for that since it might also change over time. If you see any link opportunities, let me know, even if their granular like "update a pull request". I'll make another pass at that now.

Reply:
We don’t strictly need the headings, but then again I do love “manual syncing is so 2023”